styles.css

```diff
@@ -475,12 +520,12 @@ form > input {
   border-radius: 8px;
   background-color: white;
   padding: 20px;
-  width: 65%;
+  width: 50%;
 }
 
 form > textarea {
   color: #172b4d;
-  width: 65%;
+  width: 50%;
   padding: 10px;
 }
 
```

Comment:
- [ OPTIONAL] As we are imitating the Figma Design your `font weight` should be the same as Figma one , kindly fix this so that your project will look awesome, check the `font-weight` used on Figma design :wink

This is how your font is :point_down: 
![hisfont](https://user-images.githubusercontent.com/88894525/193415771-a691d7af-e8a1-47a9-b24d-012270702e55.png)

Figma :point_down: 
![email](https://user-images.githubusercontent.com/88894525/193415799-13a05e40-cfd3-46de-830a-0ae28765354e.png)

Reply:
nobody told me earlier for background image .